src/lib/with-layout.js

```diff
@@ -79,6 +82,15 @@ export default function withLayout (Component) {
       this.setState({ toggleShowClass: !toggleShow })
     }
 
+    toggleMiniBag() {
+      const toggleShow = this.state.minibagDisplayed
+      this.setState({ minibagDisplayed: !toggleShow })
```

Comment:
Personally, I find `toggleShow` variable name to be a bit confusing, I'd consider writing this simply as:
```
this.setState({ minibagDisplayed: !this.state.minibagDisplayed })
```